scummvm_libretro.info

```diff
@@ -1,15 +1,15 @@
 # Software Information
 display_name = "ScummVM"
 authors = "SCUMMVMdev"
-supported_extensions = "0|1|2|3|4|5|6|8|25|99|101|102|455|512|scummvm|scumm|gam|z5|dat|blb|z6|ROM|001|taf|zblorb|dcp|(a)|cup|HE0|(A)|D$$|STK|z8|hex|ITK|CD1|pic|Z5|z3|blorb|ulx|DAT|cas|CGA|PIC|acd|SYS|OVL|alr|t3|gblorb|tab|AP|CRC|EXE|z4|W32|MAC|mac|WIN|003|000|exe|asl|slg|AVD|INI|SND|cat|ANG|CUP|SYS16|img|LB|TLK|MIX|RLB|#02|FNT|win|HE1|DMU|FON|SCR|MAP|TEX|HEP|DIR|DRV|a3c|GRV|CUR|CC|COD|OPT|LA0|gfx|GDA|ASK|LNG|ini|W16|SPP|bin|BND|BUN|TRS|add|HRS|DFW|BIN|STR|DR1|ALD|004|002|005|006|R02|R00|C00|D00|GAM|SCN|IDX|ogg|TXT|VB|GRA|BAT|BMV|H$$|MSG|VGA|PKD|SAV|CPS|PAK|SHP|PAT|dxr|gmp|SNG|C35|C06|WAV|wav|CAB|game|CG1|(b)|he2|he1|HE2|SYN|nl|PRC|V56|SEQ|P56|FKR|EX1|rom|CRF|LIC|$00|ALL|txt|acx|nbf|VXD|lab|LAB|ACX|mpc|msd|ADF|nib|HELLO|dsk|xfd|woz|d$$|SET|SOL|Pat|CFG|BSF|RES|CLT|LFL|SQU|RSC|SOUND|rsc|2 US|sub|cel|OVR|007|pat|MDT|CHK|EMC|ADV|voc|FDT|VQA|info|HPF|HQR|CSC|HEB|MID|LEC|QA|009|VMD|EGA|MHK|d64|prg|lfl|LZC|NL|DXR|flac|IMS|m4b|M4B|MOR|doc|jpg|HAG|AGA|BLB|PAL|PRG|CLG|CNF|ORB|BRO|bro|avi|str|PH1|DEF|sym|OUT|IN|TOC|AUD|j2|Text|CEL|AVI|1C|1c|L9|HRC|mhk|LIB|RED|PMV|SM0|SM1|RRM|CAT|CNV|GME|VOC|OGG|GERMAN|SHR|FRENCH|DNR|DSK|dnr|MMM|z4f|025|he0|V16|vga|TAB|CLU|b25c|INF|RL|mp3|SOU|SOG|HEX|mma|st|sdb|cab|MPC|MS0|IMG|ENC|C|GRP|PAR|PGM|Z|RL2|OBJ|ZFS|zfs|zip|z2|z1"
+supported_extensions = "scummvm"
```

Comment:
Note that the [ScummVM.dat database](https://github.com/libretro/libretro-database/blob/master/dat/ScummVM.dat) does allow scanning and loading ScummVM content directly from playlists.

Reply:
For info, the core info file can now be obtained from the libretro core with `make coreinfo`. Extensions are retrieved from latest ScummVM.dat, "firmwares" from ScummVM actual source.

Reply:
That's really cool! :+1:  Thanks so much.